docs/index.md

```diff
@@ -21,6 +21,16 @@ includes details that go beyond the C, C++, and Java API docs (and avoids some d
 
 This is the new home of the User Guide (since 2020 August).
 
+## ICU Site
+
+The official ICU Site is located at <https://icu.unicode.org>.
+It is the official landing page for the ICU project.
+
+Some of the pages from the ICU Site have been migrated here.
+The migrated sections and pages from the ICU Site are visible in the navigation bar of this site below the "ICU Site" section heading.
+
+These migrated versions are the canonical location for the corresponding pages (since 2022 October).
```

Comment:
The date here... I suggest we just delete this sentence, and put manual-see-over-there links on each of the Sites pages that got replaced.